account_banking_mandate/views/account_banking_mandate_view.xml

```diff
@@ -68,11 +68,15 @@
 		<field name="model">account.banking.mandate</field>
 		<field name="arch" type="xml">
 			<search string="Search Banking Mandates">

```

Comment:
yes, it's caused by the fact that there are tabs everywhere in the XML file !!! And the lines I added have spaces.